packages/utils/src/eslint-plugin/ast-utils.ts

```diff
@@ -91,6 +91,16 @@ export const angularLifecycleInterfaceKeys = objectKeys(
 );
 export const angularLifecycleMethodKeys = objectKeys(AngularLifecycleMethods);
 
```

Comment:
Would you be able to include a comment containing the URL to the lifecycle event sequence?

https://angular.io/guide/lifecycle-hooks#lifecycle-event-sequence

Reply:
Done. See below.